examples/circos/edge_color.py

```diff
@@ -0,0 +1,43 @@
+"""
+Shows different edge colors. Either categorial/ordinal or continuous
+"""
+import matplotlib.pyplot as plt
+import networkx as nx
+from nxviz.plots import CircosPlot
+
+nodelist = [('a'),
+                ('b'),
+                ('c'),
+                ('d'),
+                ('e'),
+                ('f')
+                ]
+edgelist1 = [('a', 'b', {'weight': 0.1}),
+            ('a', 'c', {'weight': 0.2}),
+            ('b', 'd', {'weight': 0.6}),
+            ('c', 'd', {'weight': 0.7}),
+            ('e', 'd', {'weight': 0.8}),
+            ('e', 'f', {'weight': 1.0}),
+            ]
```

Comment:
`pycodestyle` is also complaining about this one. I think it needs one more space before the closing bracket.